webapp/tests/karma/ts/modules/tasks/tasks.component.spec.ts

```diff
@@ -140,7 +166,7 @@ describe('TasksComponent', () => {
     expect(consoleErrorMock.args[0][0]).to.equal('Error getting tasks for all contacts');
   });
 
-  it('tasks render', async () => {
+  it('tasks render', async() => {
```

Comment:
```suggestion
  it('tasks render', async () => {
```
Before it was okay, the reason is that it's a reserved word of js